btc-transaction-helper.js

```diff
@@ -47,31 +48,79 @@ class BtcTransactionHelper{
         }
     }
 
-    async transferBtc(senderAddressInformation, receiverAddress, amountInBtc, data) {
+    /**
+     * 
+     * @param {string} address 
+     * @param {number} amountInBtc
+     * @returns 
+     */
+    async selectSpendableUTXOsFromAddress(address, amountInBtc) {
+
+        const utxos = await this.getUtxos(address);
```

Comment:
```suggestion
        const utxos = await this.nodeClient.getUtxos(address);
```

Reply:
This `getUtxos` at the btcTxHelper level is useful to be used outside of this project, as the other functions like `mine`, etc.